### PR TITLE
Add forefront DAO

### PR DESCRIPTION
**Pull request by cryptosabago**

### CODE REVIEW

File: protocols/forefront/index.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "cname": "forefront",
+  "description": "Forefront is an insight aggregator across the social token landscape.",
+  "path": "forefront",
+  "type": "snapshot",
+  "isEnabled": true,
+  "discourseForum": {
+    "url": "https://forum.forefront.news/",
+    "categoryId": "" 
+  },
+  "treasuryAddress": "0x7e9d8f07a64e363e97a648904a89fb4cd5fb94cd"
```
Comment (cryptosabago) [inline protocols/forefront/index.json:11]: 👀  actually the `contract address`...

File: protocols/forefront/index.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "cname": "forefront",
+  "description": "Forefront is an insight aggregator across the social token landscape.",
+  "path": "forefront",
+  "type": "snapshot",
+  "isEnabled": true,
+  "discourseForum": {
+    "url": "https://forum.forefront.news/",
```
Comment (cryptosabago) [inline protocols/forefront/index.json:8]: not sure about this bit, and hence not sure about the related id.

Comment (Rashmi-278) [reply protocols/forefront/index.json:8]: no worries, remove this as of now. Not all protocols have discourse

File: protocols/forefront/index.json (hunk 1 above)
Comment (Rashmi-278) [reply protocols/forefront/index.json:11]: If you are not sure about the address, can you contact their team and ask for it ?

File: protocols/forefront/overview.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+**Specification** - Proposal technical 
+
+**Outcome** - YES and No on the proposal outcome / results
+
+## Additional resources:
+
+* [Forefront](https://www.forefront.club/)
+* [Discord](https://forefront.community)
+* [Twitter](https://twitter.com/forefront__)
+* [Discourse](https://forum.forefront.news/)
```
Comment (Rashmi-278) [inline protocols/forefront/overview.md:38]: Please change this to [Forum]

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]

File: protocols/forefront/index.json (hunk 2 above)
Comment (kevinknielsen) [reply protocols/forefront/index.json]: Yeah not sure that their forum uses Discourse but there's a slight chance. I would confirm with the team

File: protocols/forefront/index.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "forefront",
+  "description": "Forefront is an insight aggregator across the social token landscape.",
+  "path": "forefront",
+  "type": "snapshot",
+  "isEnabled": true,
+  "treasuryAddress": "0x2Fb9F0ef424b24a8D293999298F392a33Fe6A8b5"
```
Comment (Rashmi-278) [inline protocols/forefront/index.json:7]: Change this to ["0x2Fb9F0ef424b24a8D293999298F392a33Fe6A8b5"]

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**